bin/ChakraCore/CMakeLists.txt

```diff
@@ -3,10 +3,6 @@ add_library (ChakraCore SHARED
     ConfigParserExternals.cpp
     TestHooks.cpp
 )
-
-# By default, don't export any symbols from this library
-# We'll manually export the relevant individual functions
-set_target_properties(ChakraCore PROPERTIES CXX_VISIBILITY_PRESET hidden)
```

Comment:
This SO inherits other static libraries with visibility=default and expect to be a proxy. If we want a particular interface is hidden, this is not the place in our case.